connect/runtime/src/main/java/org/apache/kafka/connect/runtime/Loggers.java

```diff
@@ -200,7 +198,7 @@ org.apache.logging.log4j.Logger rootLogger() {
     private void setLevel(org.apache.logging.log4j.Logger logger, Level level) {
         String loggerName = logger.getName();
```

Comment:
ditto:
```java
        var currentLevel = logger.getLevel();

        if (level.equals(currentLevel)) {
            log.debug("Skipping update for logger {} since its level is already {}", logger.getName(), level);
            return;
        }

        log.debug("Setting level of logger {} (excluding children) to {}", logger.getName(), level);
        Configurator.setLevel(logger.getName(), level);
        lastModifiedTimes.put(logger.getName(), time.milliseconds());
```